docs/asciidoc/modules/ROOT/pages/ml/bedrock.adoc

```diff
@@ -0,0 +1,225 @@
+[[aws-bedrock]]
+= AWS Bedrock procedures
+
+
+These procedures leverage the https://aws.amazon.com/bedrock/[Amazon Bedrock API].
+
+
+Here is a list of all available Aws Bedrock procedures:
+
+
+[opts=header, cols="1, 4", separator="|"]
+|===
+|name| description
+|apoc.ml.bedrock.custom(body, $config)| To create a customizable Bedrock API call
+|apoc.ml.bedrock.list($config)| To get the list of foundation or custom models
+|apoc.ml.bedrock.jurassic(body, $config)| To create an API call to `Jurassic-2` model
```

Comment:
it would be nice if we could mirror (at least in addition) the existing procedures, where we pass the model name as config parameter.
and then have at least the two for embedding generation that returns data and metadata for the embeddings
and one for chat-completion that returns the json for the chat completion

if we want to add stable diffusion i.e. image or binary generation, I suggest that we add a third type that is called `apoc.ml.bedrock.image` (and where you can pass different image models)